.github/pull_request_template.md

```diff
@@ -14,5 +14,5 @@ Please ensure your pull request adheres to the following guidelines:
 Fixes: #issue-number
 
 ```release-note
-<!-- Enter the release note text here if needed or remove this section! -->
+<!-- Enter the release note text here if needed or remove this section and use `release-note/misc` label! -->
```

Comment:
Why `release-note/misc`? That should not be always the case.

Reply:
Hm... let me double check.

Reply:
Ok, I remove the label here and the bot got back to me with https://github.com/cilium/cilium/pull/10454#issuecomment-594593680.

Reply:
The bot complains about release-note github **label**, not this section. This section can exist but if a release-note label is not set then the bot complains.

Reply:
`Enter the release note text here if the title of the PR isn't clear to users` ?

Reply:
Ah, so we actually pick the title by default? I thought removing this section completely omits the release note entry...